test/one_to_many.jl

```diff
@@ -91,15 +89,15 @@ function read_data(a::MCParticleCollection)
     @assert mcp3.generatorStatus ==3 
     @assert length(mcp3.parents)== 1
     @assert mcp3.parents[1] == mcp1
-#     @assert mcp3.parents == MCParticleCollection([mcp1])
+    @assert mcp3.parents == MCParticleCollection([mcp1])
```

Comment:
As you clarified in the email. These are indeed redundant if you check the contents explicitly above.